### DISCUSSION TITLE
Metro Context Menu strip colors

### DISCUSSION THREAD

**Discussion by rrirower** [Taiizor/ReaLTaiizor#87] [Q&A] [answered]
I'm using a metro context menu strip.  I'd like to change the forecolor for the text.  The control seems to ignore any color value I use.

How do you change the forecolor so that it is not gray?

**Comment by Taiizor** [comment, marked as answer]
You can customize Metro components the way you want using StyleManager.
Customizable sample theme file is below.

https://github.com/Taiizor/ReaLTaiizor/blob/develop/src/ReaLTaiizor/Resources/Metro/Metro%20Theme.xml
https://github.com/Taiizor/ReaLTaiizor/blob/develop/src/ReaLTaiizor/Resources/Metro/Metro%20Theme.xml#L197

How to use this theme file is shown in the example below.

https://github.com/Taiizor/ReaLTaiizor/blob/e34f1fd59bdc5c3380b52708c6b61d2d1cd127f1/demo/ReaLTaiizor.UI/ReaLTaiizor.UI/Form19.cs#L55

Or you can do it by typing the path to the theme file.
`metroStyleManager1.CustomTheme = @"C:\Custom Theme.xml";`

**Comment by rrirower** [comment]
Thanks.  I'll look into that.

**Comment by rrirower** [comment]
I finally got around to testing this.  I can now add a style manager to my form.  However, the changes that I made to the theme file do not seem to be reflected in the form, or, the context menu.

For example, I wrote the following to use my theme in the Load of the form:

`            metroStyleManager.CustomTheme = @"C:\Users\XXXXX\MetroThemeFile.xml";
`

I set the style manager field for the Metro Context menu strip to the style manager I dragged onto the form.  I changed the menu fore color to 

`<ForeColor>#000000</ForeColor>`

The text did not change.

What am I missing?

**Comment by rrirower** [comment]
I've made some progress in resolving this.  Most of my problems are due to a lack of understanding of what each item in the theme file affects.  By trial and error, I was able to learn what values affected the colors I wanted to change.  Additionally, I found that I had to change the 'RenderMode' for the context menu to "System".  I don't know if this is correct, but, it provides what I need.

You can mark this as resolved.
